pkg/sdk/random_test.go

```diff
@@ -4,10 +4,19 @@ import (
 	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance/helpers/random"
 )
 
+var (
+	invalidAccountObjectIdentifier = NewAccountObjectIdentifier(random.StringN(256))
+	invalidSchemaObjectIdentifier  = NewSchemaObjectIdentifier(random.StringN(255), random.StringN(255), random.StringN(255))
```

Comment:
I get why the upper one is invalid, but I'm not sure why this one is invalid 😕

Reply:
It's too long :D (it was just moved): https://docs.snowflake.com/en/sql-reference/identifiers-syntax

Reply:
Hmm, I thought 255 is a capacity for a "part", so individually database, schema, and schema object name are valid.